packages/ast-spec/src/legacy-fixtures/basics/fixtures/abstract-class-with-declare-properties/fixture.ts

```diff
@@ -3,7 +3,7 @@
 abstract class AbstractDeclProps {
   declare prop1: string;
   declare abstract prop2: string;
-  public declare abstract prop3: string;
+  declare public abstract prop3: string;
   declare abstract readonly prop4: string;
-  public declare abstract readonly prop5: string;
+  declare public abstract readonly prop5: string;
```

Comment:
https://prettier.io/blog/2024/11/26/3.4.0.html#print-declare-before-accessibility-in-class-properties-16731httpsgithubcomprettierprettierpull16731-by-fiskerhttpsgithubcomfisker - this and the snapshot changes are expected.